force-app/main/default/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -428,9 +440,8 @@ public with sharing class CRLP_RollupUI_SVC {
     * @param detailObjectNames list of detail or objects to rollup to query
     * @return a map of strings to string-string maps. Map<String,Map<String,String>>
     */
-
-    @AuraEnabled
```

Comment:
Thanks for spotting all of these, they definitely don't need to be `@AuraEnabled` or even `public`!